app/src/main/java/io/homeassistant/companion/android/launch/LaunchActivity.kt

```diff
@@ -100,7 +100,7 @@ class LaunchActivity : AppCompatActivity(), LaunchView {
                 val deviceName = intent.getStringExtra("DeviceName").toString()
                 val deviceTrackingEnabled = intent.getBooleanExtra("LocationTracking", false)
                 val messagingToken = getMessagingToken()
-                if (messagingToken.isNullOrBlank()) {
+                if (messagingToken.isNullOrBlank() && BuildConfig.FLAVOR == "full") {
```

Comment:
While this makes sense for the Quest where no notifications when sideloading is the default, shouldn't minimal users still get the warning about no push notifications? (at least until the local push PR is merged)

Reply:
Well minimal users never had notifications so the pop-up doesn't really make sense for them, I also think in the past we skipped this message but I cant recall. Its really intended for the full version in case we fail to register for notifications which typically happens in debug builds. For the minimal version its mentioned on F-Droid as well as our documentation that notifications are not supported. The notification section is also hidden in settings for these flavors.

https://companion.home-assistant.io/docs/core/android-flavors